File: _exercises/01_hello-llm/01_hello-llm.py
# %% [markdown]
# We'll use chatlas to interact with models available through Posit AI Pass.

# %%
import chatlas

# %% [markdown]
# The first call opens a browser so you can sign in to Posit AI Pass.

# %%
chat = chatlas.ChatPosit()
chat.chat(
    "I'm at posit::conf(2026) to learn about programming with LLMs in R and Python! "
    "Write a short social media post for me."
)

File: _exercises/01_hello-llm/01_hello-llm.R
# We'll use ellmer to interact with models provided by Posit AI.
library(ellmer)

# The first call opens a browser so you can sign in to Posit AI.
chat <- chat_posit()
chat$chat(
  "I'm at posit::conf(2026) to learn about programming with LLMs in R and Python!",
  "Write a short social media post for me."
)
